Resources/Prototypes/Entities/Structures/Walls/base_structurewalls.yml

```diff
@@ -12,6 +12,7 @@
   - type: Tag
     tags:
     - Wall
+    - RCDDeconstructWhitelist
```

Comment:
dont add that to base wall

Reply:
Why and how else?

Reply:
After prototype composition is merged via that, or just add it to walls as needed.

Reply:
add it to the children walls